src/schemes/fluid/implicit_incompressible_sph/system.jl

```diff
@@ -563,3 +595,8 @@ function calculate_sum_term(system, neighbor_system::AbstractBoundarySystem, par
     # m_j * sum_k d_ik * p_k * ∇W_ij
     return m_j * dot(sum_dik_pk, grad_kernel)
 end
+
+function calculate_source_term(system::ImplicitIncompressibleSPHSystem, particle)
```

Comment:
```suggestion
function iisph_source_term(system::ImplicitIncompressibleSPHSystem, particle)
```
I think this name is better here. The other `calculate` functions update the system, which is not the case here. We don't want to call it just `source_term` because this function already exists outside IISPH.